stream-chat-android/src/main/java/com/getstream/sdk/chat/view/MessageListView.kt

```diff
@@ -242,6 +241,9 @@ public class MessageListView : ConstraintLayout, IMessageListView {
         if (attr != null) {
             configureAttributes(attr)
         }
+        loadMoreListener = EndlessScrollListener(loadMoreThreshold) {
+            endRegionReachedHandler()
+        }
```

Comment:
I think we could move this into `configureAttributes` to get rid of the `loadMoreThreshold` property (in both of these cases)